pom.xml

```
@@ -40,10 +40,10 @@
 
     <properties>
         <gravitee-bom.version>2.6</gravitee-bom.version>
-        <gravitee-node.version>1.25.0</gravitee-node.version>
+        <gravitee-node.version>2.0.0-bumpRxJava3-SNAPSHOT</gravitee-node.version>
```

Could you please set it to the pre-released version ?
```suggestion
        <gravitee-node.version>2.0.0-alpha.3</gravitee-node.version>
```